ext/node/polyfills/internal/fs/handle.ts

```diff
@@ -24,6 +24,11 @@ export class FileHandle extends EventEmitter {
   ): Promise<string | Buffer> {
     return promises.readFile(this, opt);
   }
+
+  close(): Promise<void> {
+    // Note that Deno.close is not async
+    return Promise.resolve(Deno.close(this.fd))
```

Comment:
Looks like this just needs a semicolon to pass the linter (run tools/format.js) and we're good to go

Reply:
Oops..